packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -38,4 +70,177 @@ Content-Type: application/octet-stream;
   @post
   @route("/mixed-parts")
   op basic(@header contentType: "multipart/form-data", @body body: MultiPartRequest): NoContentResponse;
+
+  @scenario
+  @scenarioDoc("""
+Expect request:
+```
+POST /upload HTTP/1.1
+Content-Length: 428
+Content-Type: multipart/form-data; boundary=abcde12345
+
+--abcde12345
+Content-Disposition: form-data; name="id"
+Content-Type: text/plain
```

Comment:
Again, is this optional or required?

Reply:
similar with https://github.com/Azure/cadl-ranch/pull/460#discussion_r1434684155